src/main/java/it/pagopa/ecommerce/payment/methods/application/v2/PaymentMethodService.java

```diff
@@ -154,6 +154,25 @@ private CalculateFeeResponseDto bundleOptionToResponse(
                                 .taxPayerFee(t.getTaxPayerFee())
                                 .touchpoint(t.getTouchpoint())
                                 .pspBusinessName(t.getPspBusinessName())
+                ).sorted(
```

Comment:
here we have a requisite for which psp with same fees should be shuffled.
no shuffle operation is performed here based on the fact that an external dependency perform it for us.
btw I would put a shuffle mechanism here too just to be sure that psps with same fees are shuffled, independently from an external dependency response ordering, enforcing the requirement here to be satisfied also if the GEC remove shuffle by it's side